static/css/blog-docs.less

```diff
@@ -414,6 +414,56 @@
     - cfgov-blog-docs
 */
 
+@gutter__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@gutter-half__em: @gutter__em/2;
+
+.expandable__filters {
+    border: 1px solid @gray-50;
+    margin: unit(45px / @base-font-size-px, em) 0 @gutter__em 0;
+
+    &.expandable__padded, .expandable_content {
+        background:  @gray-5;
+    }
+
+    .expandable_content {
+        margin: 0 @gutter__em @gutter__em;
+
+        .respond-to-max(@mobile-max {
+            margin: 0 @gutter-half__em @gutter-half__em;
+        });
+    }
+
+    .expandable_header {
+        padding: @gutter__em/3 @gutter__em;
+
+        .respond-to-max(@mobile-max {
+            padding:  @gutter__em/3 @gutter-half__em;
+        });
+    }
+
+    .expandable_header-right {
+        .respond-to-max(@mobile-max {
+            margin-left: unit(20px / @base-font-size-px, em);
+         });
+    }
+
+    .lt-ie9 {
+        .ie-date_format {
+            display: inline-block;
+        }
+    }
+
+    .ie-date_format {
+        display: none;
```

Comment:
No, I want to target just that specific element as opposed those that had the .u-visually-hidden class. It's just being used to add 'mm/dd/yyyy' labels for IE